CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+## v0.73.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.73.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.73.0) and the [opentelemetry-collector-contrib v0.73.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.73.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- [Add java zc options to linux installer script](https://github.com/signalfx/splunk-otel-collector/pull/2718)
+- [Update installer script for generate_service_name option](https://github.com/signalfx/splunk-otel-collector/pull/2730)
+- [Build experimental linux arm64 agent-bundle](https://github.com/signalfx/splunk-otel-collector/pull/2671)
+
```

Comment:
I believe we should add references to the addition of the zc options for Ansible, Puppet, Chef and Salt too.

Reply:
How about something like:
```suggestion
- Added profiling and JVM metrics options for zero configuration auto instrumentation of Java apps (Linux only):
  - [Installer script](https://github.com/signalfx/splunk-otel-collector/pull/2718)
  - [Ansible v0.16.0](https://github.com/signalfx/splunk-otel-collector/pull/2729)
  - [Chef v0.5.0](https://github.com/signalfx/splunk-otel-collector/pull/2733)
  - [Puppet v0.9.0](https://github.com/signalfx/splunk-otel-collector/pull/2734)
  - [Salt](https://github.com/signalfx/splunk-otel-collector/pull/2735)
```

Reply:
Nice, +1. Thanks Jeff!